docs/master/api-reference/directives.md

```diff
@@ -2173,6 +2173,34 @@ scalar DateTime
     @scalar(class: "Nuwave\\Lighthouse\\Schema\\Types\\Scalars\\DateTime")
 ```
 
+## @scope
+
+Adds a scope to the query builder through a graphql argument.
```

Comment:
To reduce duplication/maintenance, we can remove the `## Definition` section and just pull that up here.

That way, we can put a nice, explicit description in the directive definition and have that available everywhere.